Core/HLE/sceJpeg.cpp

```diff
@@ -139,12 +139,19 @@ static int __DecodeJpeg(u32 jpegAddr, int jpegSize, u32 imageAddr) {
 	if (actual_components == 3) {
 			u24_be *imageBuffer = (u24_be*)jpegBuf;
 			u32 *abgr = (u32*)Memory::GetPointer(imageAddr);
+			int pspWidth;
```

Comment:
Hmm, this isn't initialized, what if the width and height are crazy values, like 4096?  Not that it should happen, but good to handle error scenarios.

-[Unknown]